packages/desktop/components/ManaBox.svelte

```diff
@@ -41,7 +42,7 @@
             requiredTxManaCost = Number(requiredManaForTransaction ?? 0)
 
             const slotsRemaining = Number(splittedError.reverse()[0].replace('`', ''))
-            if (slotsRemaining) {
+            if (slotsRemaining && !secondsRemaining) {
```

Comment:
I think this is more readable.
```suggestion
            if (slotsRemaining && secondsRemaining === 0) {
```